services/report/raw_upload_processor.py

```diff
@@ -124,7 +127,7 @@ def process_raw_upload(
             log.info(
                 "Customer is using joined=False feature", extra=dict(flag_used=flag)
             )
-            joined = False
+            joined = False  # TODO: make this work for parallel
```

Comment:
I have no idea what this `joined` feature is. This is an example of what hopefully could've been surfaced from this verification experiment. 

I feel like there very well a lot of small features like this that I'm not able to reproduce/know of locally.